python/lsst/pipe/base/pipeline.py

```diff
@@ -520,8 +530,8 @@ def addTask(self, task: Union[PipelineTask, str], label: str) -> None:
             # be defined without label which is not acceptable, use task
             # _DefaultName in that case
             if isinstance(task, str):
-                task = doImport(task)
-            label = task._DefaultName
+                task_class = doImport(task)
```

Comment:
`doImportType` but also, how does mypy know that `task_class._DefaultName` going to work without an assert?